package.json

```diff
@@ -50,16 +50,21 @@
     "eslint": "6.8.0",
     "jest": "29.4.2",
     "lint-staged": "13.1.2",
+    "memory-fs": "0.5.0",
     "prettier": "2.8.4",
     "typescript": "4.9.5"
   },
+  "optionalDependencies": {
+    "fast-crc32c": "1.0.7"
+  },
```

Comment:
Do we need this dependency? https://github.com/xlts-dev/verdaccio-google-cloud-storage/pull/17#discussion_r1128544600

Reply:
Nope :smiley:
I guess this is a git rebase issue. Removed.